docs/components/accordion/accordion.njk

```diff
@@ -170,7 +170,7 @@ An Accordion is a vertically stacked set of headings that each control the visib
 
 <h4>Header</h4>
 
-<table class="jazz-table jazz-table--bordered">
```

Comment:
Why are you removing the bordered style the JIRA only specifies making the tables transparent?

Reply:
Oh I see now you included changes from the table styling JIRA in this commit